client/src/components/layout/UserBuildsList.js

```diff
@@ -1,10 +1,16 @@
-import React, { Fragment } from "react";
+import React from "react";
 import UserBuildTile from "./BuildTile"
+import DeleteBuildButton from "./DeleteBuildButton";
 
-const UserBuildsList = ({ buildsList }) => {
+const UserBuildsList = ({ buildsList, currentUserData, setUserData }) => {
     const userBuildTiles = buildsList.map((build) => {
         return (
-            <UserBuildTile key={build.id} build={build} />
+            <li key={build.id}>
+                <UserBuildTile build={build} currentUserData={currentUserData} setUserData={setUserData} />
+                <div>
+                    <DeleteBuildButton dataToDelete={build.id} currentUserData={currentUserData} setUserData={setUserData} />
```

Comment:
if we want each build tile to have a delete button, render the deletebutton inside the build tile to reduce all the JSX returned from this map